string class_name
---
string message
bool success
